src/sentry/snuba/query_subscription_consumer.py

```diff
@@ -38,6 +62,153 @@ def inner(func: TQuerySubscriptionCallable) -> TQuerySubscriptionCallable:
     return inner
 
 
+def parse_message_value(value: str) -> Dict[str, Any]:
```

Comment:
I didn't read through this diff, assuming it's mostly just moved code

Reply:
Yes this function is untouched, just moved.